docs/modules/ROOT/pages/explanations/decisions/multi-instance-argocd.adoc

```diff
@@ -0,0 +1,50 @@
+= Multiple ArgoCD Instances
+
+== Problem
+
+If a customer also want's to use ArgoCD for their deployment, he currently has to install ArgoCD with the same version we use and not install any CRDs.
+This process needs synchronization between us and the customer, if there is a new version of ArgoCD to be rolled out.
+
+=== Goals
+
+* We can deploy multiple instances of ArgoCD
+* No synchronization needed between us and customer, if an update needs to be rolled out
+
```

Comment:
```suggestion

=== Non-Goals

* Allow multiple different versions of ArgoCD to be installed on the same cluster
* Provide VSHN-managed ArgoCD instances to the customer
```

Is the second bullet actually a non-goal? In my opinion it is, since this should explicitly be a goal if we want to optionally offer that we manage the customer's instance config for them.